src/PreferencesDialog.ui

```diff
@@ -555,159 +555,152 @@
       <attribute name="title">
        <string>&amp;Database</string>
       </attribute>
-      <layout class="QFormLayout" name="formLayout">
-       <property name="fieldGrowthPolicy">
-        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
-       </property>
-       <item row="0" column="0">
-        <widget class="QLabel" name="label">
-         <property name="text">
-          <string>Database &amp;encoding</string>
+      <layout class="QVBoxLayout" name="verticalLayout_10">
+       <item>
+        <layout class="QFormLayout" name="formLayout">
+         <property name="fieldGrowthPolicy">
+          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
-         <property name="buddy">
-          <cstring>encodingComboBox</cstring>
+         <property name="leftMargin">
+          <number>9</number>
          </property>
-        </widget>
+         <property name="topMargin">
+          <number>9</number>
+         </property>
+         <property name="rightMargin">
+          <number>9</number>
+         </property>
+         <property name="bottomMargin">
+          <number>9</number>
```

Comment:
Can you remove these four margins of size 9? We don't have them in the other layouts, so switching between the tabs looks inconsistent. Saying this also because we should avoid explicit sizes in pixels wherever it's possible because it can give unexpected results for people with HiDPI monitors.